test/e2e/page-objects/pages/permission/permission-list-page.ts

```diff
@@ -11,6 +11,8 @@ class PermissionListPage {
 
   private readonly permissionsPage = '[data-testid="permissions-page"]';
 
+  private readonly backButton = '[data-testid="permissions-page-back"]';
+
```

Comment:
nit(non-blocking): it would be nice to have the selectors ordered alphabetically